tezos/context/src/serialize/persistent.rs

```diff
@@ -1684,35 +1690,36 @@ mod tests {
 
     #[test]
     fn test_hash_id() {
+        let mut repo = Persistent::try_new(None).expect("failed to create context");
         let mut output = Vec::with_capacity(10);
         let mut stats = Default::default();
 
-        let number = 10101;
+        let number = HashId::new(10101).unwrap();
```

Comment:
`HashId::try_new` would be a better name.

Reply:
hmm not sure about that, `HashId::new` is only a call to `NonZeroU64::new`:
https://github.com/tezedge/tezedge/blob/develop/tezos/context/src/kv_store/mod.rs#L60
`NonZeroU64::new` is not `NonZeroU64::try_new`

Reply:
You are right. I always assumed `try_new` was expected when failure was possible.